### PR TITLE
Linking accompanying blog post

### PR DESCRIPTION
**Pull request by pdolega**
I thought that it might give users some context.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,9 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/VirtusLab/render "Go Report Card")](https://goreportcard.com/report/github.com/VirtusLab/render)
 
 Universal file renderer based on [go-template](https://golang.org/pkg/text/template/) 
-and [Sprig](http://masterminds.github.io/sprig/) functions.
+and [Sprig](http://masterminds.github.io/sprig/) functions. 
+
+If you want to read more about the underlying reason for creating this library and our initial use of it, take a look at this [blog post](https://medium.com/virtuslab/helm-alternative-d6568aa9d40b) (hint: Kubernetes config files rendering).
```
Comment (pawelprazak) [inline README.md]: that is one of the use cases, wouldn't want to attach a "k8s label" too much, it actually can be used for any text file :)

so I'd rephrase it to something along the lines of: "If you are interested in one of the use cases, take a look at..."

Comment (placydo) [reply README.md]: too late I have already merged it

Comment (pdolega) [reply README.md]: No worries - will fix in 2 mins

Comment (pdolega) [reply README.md]: https://github.com/VirtusLab/render/pull/2

Keep in mind you can edit to whatever you want *in place* (in a PR, from web UI).
